server/application-server/src/main/resources/db/master.xml

```diff
@@ -5,6 +5,7 @@
     xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
 
-    <include file="initial-schema.xml" relativeToChangelogFile="true"/>
+    <include file="./changelog/0000000000000_initial_schema.xml" relativeToChangelogFile="true"/>
```

Comment:
This will break the production db, but I guess that is fine. I can maybe surgically edit the changelog entry.

Reply:
We could keep the inital_schema in the top folder if that makes it easier? 
I tried to stick with Artemis's folder placement and naming schema here, as discussed in the last Hephaestus meeting.